snmp/datadog_checks/snmp/data/default_profiles/infoblox.yaml

```diff
@@ -0,0 +1,8 @@
+extends:
+  - _base.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "infoblox"
```

Comment:
since this is a generic infoblox profile, should we add 
```
sysobjectid:
  - 1.3.6.1.4.1.7779.*
``` 
?

Reply:
I'm not sure if there is a need to add it right now since we don't have much things here beside the metadata.

Reply:
I can also inline / move the vendor metadata to `infoblox-ipam.yaml` if needed since we only have 1 profile for now.

Reply:
could be a good idea to reduce the number of profiles

Reply:
@NouhaManai96 

👍 infoblox.yaml removed here https://github.com/DataDog/integrations-core/pull/15354/commits/8020e07386189fd6865a4f0c5de8f0e106ee078b